docker-compose.yml

```diff
@@ -21,11 +22,12 @@ services:
       context: .
       dockerfile: ./docker/dev/Dockerfile
     command: ["gunicorn", "busy_beaver:app", "-b", "0.0.0.0:5100", "--reload", "--timeout",  "100000"]
+#    env_file: ./db.env   #TODO: explicitly defining the name for an `env config` doesn't seem to work!?  :(
```

Comment:
Yes, doesn't look like this is possible

https://docs.docker.com/compose/env-file/

Reply:
Nevermind

https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option